org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/internal/ui/text/correction/JavadocTagsBaseSubProcessor.java

```diff
@@ -267,14 +278,18 @@ public void addMissingJavadocCommentProposals(IInvocationContextCore context, IP
 				}
 				if (comment != null) {
 					String label= CorrectionMessages.JavadocTagsSubProcessor_addjavadoc_field_description;
-					proposals.add(addJavadocCommentProposal(label, cu, IProposalRelevance.ADD_JAVADOC_FIELD, declaration.getStartPosition(), comment));
+					T p = addJavadocCommentProposal(label, cu, IProposalRelevance.ADD_JAVADOC_FIELD, declaration.getStartPosition(), comment);
+					if( p != null )
```

Comment:
global comment - stick to existing if formatting.  Have if (expression) rather than jamming open parenthesis against if and adding spaces around expression